astro/tests/integration_test.go

```diff
@@ -272,12 +268,20 @@ func TestVersionWithLdflags(t *testing.T) {
 
 func TestPlanErorrsWithoutConfig(t *testing.T) {
 	dir, err := ioutil.TempDir("/tmp", "astro-tests")
-	if err != nil {
-		panic(err)
-	}
+	require.NoError(t, err)
 	defer os.RemoveAll(dir)
```

Comment:
Defer should go one line above.

(You just gave me the same feedback 😄 )

Reply:
Yep, changed it and combined tests.